Parser/Parser.php

```diff
@@ -35,15 +44,24 @@ public function __construct()
      *
      * @param string $rootDirectoy
      * @param OutputInterface|null $output
-     * @return array
+     * @param string[] $filterDirectories
+     * @return CallToContainer[]
      */
-    public function parseDirectory(string $rootDirectoy, OutputInterface $output = null): array
+    public function parseDirectory(string $rootDirectoy, OutputInterface $output = null, $filterDirectories = ['chameleon-system/upgrade-helper']): array
```

Comment:
A short explanatin of what `filterDirectories` does would help the next person reading it (e.g. it matches the full path, not a regex, matches are excluded from the result)

Reply:
done

Reply:
:ok:

Reply:
:ok: